src/common/config/defaultPreferences.js

```diff
@@ -15,7 +15,8 @@ const defaultPreferences = {
   },
   showUnreadBadge: true,
   useSpellChecker: true,
-  spellCheckerLocale: 'en-US'
+  spellCheckerLocale: 'en-US',
+  autoStart: false
```

Comment:
Currently, `autoStart` is `true`, I believe?

Reply:
I have thought the default is false though, the document says it's true.
https://docs.mattermost.com/help/apps/desktop-guide.html#app-options

Should we enable this?

Reply:
I think we should keep the default setting at what it's now. If the default is `false` in the code, let's keep it `false` and I can just update the documentation.

Reply:
I reinstalled v3.7.1 after removing config.json. The default is false.

Reply:
Sounds good, let's keep it false. Updated doc https://github.com/mattermost/docs/commit/5892490d7ce36e80acac7fdfdfada261be5b5e56